docs/tutorial.md

```diff
@@ -0,0 +1,317 @@
+# IVSPlayer component usage guide
+
+Let's create a full working example that implements advanced `IVSPlayer` component with the control buttons and time slider.
+Our example will contain:
+
+- `IVSPlayer` component with bunch of properties like `autoplay, streamUrl, volume` or `quality`
+- Few event listeners that will help us react on specific Player events like `onProgress` or `onDurationChange`
+- Control buttons for pausing, playing and muting the player.
+- Time slider to control the current position of the video.
+
+## Rendering `IVSPlayer` component in your app
+
+At the very beginning, let's create a screen where the Player will be placed.
+To render the Player on your screen just use [`IVSPlayer`](./ivs-player-reference.md) component wherever you need it.
+
+> Please note that player dimensions adjust to its parent component.
+
+```jsx
+import IVSPlayer from 'amazon-ivs-react-native';
+
+export default function App() {
+  return (
+    <View style={styles.container}>
+      <IVSPlayer />
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    height: 300,
+  },
+});
+```
+
+To make the Player work we need to at least add `streamUrl` prop which is gonna load the video or a live stream.
```

Comment:
`we need to at least add the streamUrl prop which loads the video or live stream`